templates/swift/Sources/Services/Realtime.swift.twig

```diff
@@ -137,7 +139,7 @@ open class Realtime : Service {
             let subsWithChannel = activeSubscriptions.filter { callback in
                 return callback.value.channels.contains(channel)
             }
-            return subsWithChannel.isEmpty
+            return !subsWithChannel.isEmpty
```

Comment:
Android code as a reference -> https://github.com/appwrite/sdk-generator/blob/master/templates/android/library/src/main/java/io/package/services/Realtime.kt.twig#L130

Functions are similar, but since Swift code is filtering statement should have been different than in Android in order to properly clear channels.